### ISSUE TITLE
Validation errors should honour the localization name for the model

### ISSUE THREAD

**Issue by stuzart** [seek4science/seek#473]
The validation error should use the correct term, rather than the raw model name. For example, when missing to select a Project, but the project has been renamed Team, it should say words to the effect of "Team is required". It currently says Project regardless of the localisation change

**Comment by fbacall** [comment]
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models

**Comment by fbacall** [comment]
We could use YAML anchors for this...

e.g.
```yaml
en:
  investigation: &investigation "Investigation"
  study: &study "Study"
  assay: &assay "Assay"

  activerecord:
    models:
      investigation: *investigation
      study: *study
      assay: *assay
```

**Comment by abecam** [comment]
So there were "aliases" already, in seekMain:

`  #Need to load defaut_locale file for internationalization used in Inflector below
  #coz this file is loaded at a later point
  I18n.load_path << File.join(File.dirname(__FILE__), "../locales/en.yml")
  #these inflections are put here, because the config variables are just loaded after the propagation
  ActiveSupport::Inflector.inflections do |inflect|
    inflect.human 'Assay', I18n.t('assays.assay')
    inflect.human 'assay', I18n.t('assays.assay')
    inflect.human 'Sop', I18n.t('sop')
    inflect.human 'sop', I18n.t('sop')
    inflect.human 'Presentation', I18n.t('presentation')
    inflect.human 'presentation', I18n.t('presentation')
    inflect.human 'DataFile', I18n.t('data_file')
    inflect.human 'data_file', I18n.t('data_file')
    inflect.human 'Investigation', I18n.t('investigation')
    inflect.human 'investigation', I18n.t('investigation')
    inflect.human 'Study', I18n.t('study')
    inflect.human 'study', I18n.t('study')
    inflect.human 'Model', I18n.t('model')
    inflect.human 'model', I18n.t('model')
    inflect.human 'Event', I18n.t('event')
    inflect.human 'event', I18n.t('event')
    inflect.human 'Project', I18n.t('project')
    inflect.human 'project', I18n.t('project')
  end`

**Comment by abecam** [comment]
Fixed by #1208
